app/presenters/vendor_api/single_application_presenter.rb

```diff
@@ -215,5 +215,20 @@ def hesa_itt_data
         ethnicity: '10',
       }
     end
+
+    def work_history_breaks
+      if FeatureFlag.active?('work_breaks')
+        breaks = application_form.application_work_history_breaks.inject('') do |explanation, work_break|
+          start_date = work_break.start_date.to_s(:month_and_year)
+          end_date = work_break.end_date.to_s(:month_and_year)
+
+          explanation << "#{start_date} to #{end_date}: #{work_break.reason}\n\n"
+        end
```

Comment:
Alternatively:

```ruby
breaks = application_form.application_work_history_breaks.map do |work_break|
  start_date = work_break.start_date.to_s(:month_and_year)
  end_date = work_break.end_date.to_s(:month_and_year)

  "#{start_date} to #{end_date}: #{work_break.reason}"
end.join('\n\n')
```

No more need for `.strip` after!

Reply:
Rubocop shouted at me for chaining a method call at the end of `do..end`. So did dis instead:

```ruby
breaks = application_form.application_work_history_breaks.map do |work_break|
  start_date = work_break.start_date.to_s(:month_and_year)
  end_date = work_break.end_date.to_s(:month_and_year)

  "#{start_date} to #{end_date}: #{work_break.reason}"
end

breaks.join("\n\n")
```

Reply:
Silly rubocop!